packages/@best/frontend/server/auth.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+*/
+import { Request, Response, NextFunction } from "express";
+import jwt from 'jsonwebtoken';
+
+const TOKEN_SECRET = process.env.TOKEN_SECRET as string;
+const REVOKED_TOKENS = (process.env.REVOKED_TOKENS || "").split("\n");
+
+/**
+ * Checks if the provided token has been revoked based on the revocation list provided
+ * in process.env.REVOKED_TOKENS environmental variable
+ * @param token The token to check if it has been revoked.
+ * @returns true if token is revoked, otherwise false
+ */
+function isRevoked(token: string): boolean {
+    return REVOKED_TOKENS.includes(token);
+}
+
+/**
+ * Function that verifies the token provided in the request header
+ * and on successful authorization it will call the next function on the chain.
+ * The secret token must be provided via process.env.TOKEN_SECRET
+ */
+export function authorizeRequest(req: Request, res: Response, next: NextFunction): void {
+    const { authorization: authHeader } = req.headers;
+    const token = authHeader && authHeader.split(' ')[1]
```

Comment:
@dbajric Should we check if `Bearer` auth is used? (e.g.: `authHeader.startsWith('Bearer ')`).

Reply:
We don't have to as the first part of the header is there to indicate the scheme and we only support one type.

Reply:
But I can add it just to make it more explicit.

Reply:
@alrra - updated the header verification.